src/MCreatorLink.cpp

```diff
@@ -79,7 +79,10 @@ void MCreatorLinkImpl::loop() {
         analogWrite(pin, val);
       }
     } else if (command.equals("msg")) {
-      event(command, data);
+    	String tempComm = data.substring(0, data.indexOf(":"));
```

Comment:
This need indexOf check as it can be -1, and in this case only use command, with empty string for data

Reply:
In case there is no data passed, tempData will be empty automatically, I don't have to do any if statements